buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/UiTests.kt

```diff
@@ -54,6 +54,8 @@ fun Project.configureUiTests() {
         }
       }
 
+      environment("LOG_EXPORT", "${System.getProperty("user.home")}/.ideprobe-uitests/idea-logs")
```

Comment:
While you're here, pls extract `${System.getProperty("user.home")}/.ideprobe-uitests/` into a variable